docs/user-guides/desktop/index.md

```diff
@@ -0,0 +1,124 @@
+# Coder Desktop
+
+Use Coder Desktop to work on your workspaces as though they're on your LAN, no
+port-forwarding required.
+
+## Install Coder Desktop
+
+<div class="tabs">
+
+You can install Coder Desktop on macOS or Windows.
+
+### macOS
+
+1. Use [Homebrew](https://brew.sh/) to install Coder Desktop:
+
+   ```shell
+   brew install --cask coder/coder/coder-desktop
+   ```
+
+1. Open Coder Desktop from the Applications directory and when macOS asks if you want to open it, select **Open**.
+
+1. The application is treated as a VPN. macOS will prompt you to confirm with:
+
+   **"Coder Desktop" would like to use a new network extension**
+
+   Select **Open System Settings**.
+
+1. In the **Network Extensions** system settings, enable the Coder Desktop extension.
+
+1. Continue to the [configuration section](#configure).
+
+### Windows
+
+1. Download the latest `CoderDesktop` executable (`.exe`) from the [coder-desktop-windows repository](https://github.com/coder/coder-desktop-windows/).
+
+   Choose the architecture that fits your Windows system, ARM or x64.
+
+1. Open the `.exe` files, acknowledge the license terms and conditions, and select **Install**.
+
+1. Windows asks if you "want to allow this app from an unknown publisher to make changes to your device?"
+
+   Select **Yes**.
+
+1. The Coder Desktop installation creates a system restore point.
+   Depending on your system, this step might take a while.
+
+1. If .NET is not already installed, the installation might prompt you with the **Windows Desktop Runtime** installation.
+
+   In that installation window, select **Install**. Select **Close** when the Runtime installation completes.
+
+1. When the Coder Desktop installation completes, select **Close**.
+
+1. Find and open **Coder Desktop** from your Start Menu.
+
+1. Some systems require an additional Windows App Runtime.
```

Comment:
This should probably use similar language to the .NET step. Pretty much all users will get prompted unless they have the windows app sdk runtime installed.